Update script-storage.md (#603)
OfficeDev/office-scripts-docs: docs/overview/script-storage.md

The default behavior is that Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.

> Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel. The move button of the script renaming callout moves a script to a new location. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.

Reviewer: Suggestion based on the button rename. This also uses less UI jargon and tries to make the process more relatable.
```suggestion
Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel. The **Move** button shown when renaming a script moves the script to a new location. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.
```

To load a script from SharePoint, go to **All scripts** and select **View more scripts** at the bottom of the list. This brings up a file picker where you can choose **.osts** files from any SharePoint site to which you have access. Note that scripts from SharePoint that you've already opened will be displayed in the list of recent scripts.

> To move a script to SharePoint, open a script in the editor and click on the script name. This will open a callout. In the callout, click the move button. This opens a file picker where you can select folders in your SharePoint site. Saving to a new location moves the script to that location. This can take up to a minute.

Reviewer: Similarly, this suggestion is to try and make the UI choices more relatable to the user.

```suggestion
To move a script to SharePoint, open a script in the **Code Editor** and click on the script name, as if you're renaming it. In the callout, click the **Move** button. This opens a file picker. Select the destination folder in your SharePoint site. Moving the script to the new location can take up to a minute.
```
